### PR TITLE
nitch: add page

### LINKED ISSUE #10903: page request: nitch
### Command description

nitch is a small and incredibly fast system fetch written fully in nim. 

### Command details

nitch

### OS

- [ ] Android
- [X] Linux
- [ ] macOS (osx)
- [ ] sunOS
- [ ] Windows

### PR DESCRIPTION
**Pull request by npv12**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

fixes #10903

### CODE REVIEW

File: pages/linux/nitch.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# nitch
+
+> Nitch is a small and incredibly fast system fetch written fully.
+> More information: <https://github.com/ssleert/nitch>.
+
+- Return the system info like hostname, kernel, uptime, etc.:
```
Comment (kbdharun) [inline pages/linux/nitch.md:6]:
```suggestion
- Return the system information like hostname, kernel, uptime, etc.:
```

File: pages/linux/nitch.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`nitch -h`
+
+- Use -v flag to see the version:
+
+`nitch -v`
+
+- Use -f flag to fetch the system info:
+
+`nitch -f`
```
Comment (kbdharun) [inline pages/linux/nitch.md:20]:
```suggestion
- Fetch the system information:

`nitch -f`

- Display help:

`nitch -h`

- Display version:

`nitch -v`
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

Note: We use generic descriptions like `Display help:` and `Display version:` for help and version examples.

**Comment by npv12** [comment]
> Welcome to tldr and thanks for your contribution. I have some suggestions for this page.
> 
> Note: We use generic descriptions like `Display help:` and `Display version:` for help and version examples. Also we place these examples at the last part of the page with help being the last second command and version being the last one.

Thanks! Updated my MR

**Review by kbdharun — APPROVED** [review]

File: pages/linux/nitch.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+> Nitch is a small and incredibly fast system fetch written fully.
+> More information: <https://github.com/ssleert/nitch>.
+
+- Return the system information like hostname, kernel, uptime, etc.:
+
+`nitch`
+
+- Fetch the system information:
+
+`nitch -f`
```
Comment (acuteenvy) [inline pages/linux/nitch.md:12]: This is exactly the same as `nitch` without arguments.
```suggestion
```

File: pages/linux/nitch.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Display help:
+
+`nitch -h`
+
+- Display version:
+
+`nitch -v`
```
Comment (acuteenvy) [inline pages/linux/nitch.md:20]: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax
```suggestion
- Display help:

`nitch --help`

- Display version:

`nitch --version`
```

File: pages/linux/nitch.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+> More information: <https://github.com/ssleert/nitch>.
+
+- Display system information (hostname, kernel, uptime, etc.):
+
+`nitch`
+
+- Exactly the same as nitch without arguments:
+
+`nitch -f`
+
```
Comment (gutjuri) [inline pages/linux/nitch.md:13]:
```suggestion
```

I'd delete this, as `-f` does not have an effect.

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thanks for your contribution. However, I'd delete the description of the -f option, as it does not seem to have an effect  ([it's not even implemented](https://github.com/ssleert/nitch/blob/main/src/flags/argParser.nim)). There also seems to be a `-a/--no-ascii` option, which is undocumented. I'm not sure whether it does anything, as I'm not familiar with the program. You might be a better judge to decide whether or not to include this option in the tldr page.

**Comment by npv12** [comment]
> Thanks for your contribution. However, I'd delete the description of the -f option, as it does not seem to have an effect ([it's not even implemented](https://github.com/ssleert/nitch/blob/main/src/flags/argParser.nim)). There also seems to be a `-a/--no-ascii` option, which is undocumented. I'm not sure whether it does anything, as I'm not familiar with the program. You might be a better judge to decide whether or not to include this option in the tldr page.

--no-ascii has no effect

**Review by gutjuri — APPROVED** [review]

**Pull request merged**